packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/components/SubjectPreview/SubjectPreview.js

```diff
@@ -0,0 +1,83 @@
+import counterpart from 'counterpart'
+import { array, bool, number, shape, string } from 'prop-types'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Anchor, Box } from 'grommet'
+import { Media } from '@zooniverse/react-components'
+import { CollectionsButton, FavouritesButton, TalkLink } from './components'
```

Comment:
British or American spelling of "favourite"? I don't mind which, as long as it's consistent

Reply:
It's `Favourite` in the classifier components. 🤷‍♂

Reply:
I think `favorite` is used elsewhere in this PR? (might be from the API, mind)

Reply:
Yes, Panoptes uses `favorite`.

Reply:
I can't see a `favorite` flag on recents returned from Panoptes, so I've changed this to use `favourite`. We can revisit that if the API does start indicating which subjects are in your favourites.

Reply:
Subjects do have `subject.favorite` though. 🤔 I might have been too hasty with that change.